### DISCUSSION TITLE
Updating component properties via "@this" Blade directive doesn't seem to work in v3

### DISCUSSION THREAD

**Discussion by jj15asmr** [livewire/livewire#6627] [1. Help] [answered]
Hello!

I'm currently in the process of upgrading from Livewire v2 to v3, in my component I have a ``start_date`` and ``end_date`` property, in the view, I am utilizing the "Date Range Picker" jQuery plugin and the ``@this`` Blade directive to update these properties when the picker's start and end dates are changed:

```js
$('#search-by-date').daterangepicker({
    opens: 'left'
}, function (start, end) {
    // Update our Livewire component props
    @this.start_date = start.format('YYYY-MM-DD');
    @this.end_date = end.format('YYYY-MM-DD');
});
```

This worked as expected in v2 but doesn't seem to in v3. No server request is sent, it will update however if I perform another action (such as picking an option from a ``select`` box).

Testing it via the browser console as so does return the updated value, but still makes no network request:

```js
window.Livewire.find('xxxxxxxxxxxxxxxxxxxx').start_date = '2023-09-01'
> '2023-09-01'
```

Trying the following code does work though and makes a network request:

```js
window.Livewire.find('xxxxxxxxxxxxxxxxxxxx').set('start_date', '2023-09-01')
```

Is this intentional, such as how ``wire:model`` is deferred by default now?

Thanks :)

**Comment by jj15asmr** [comment, marked as answer]
It seems that this change is intentional so using ``@this.set()`` (https://livewire.laravel.com/docs/properties#manipulating-properties) is the proper way to go now.

**Reply by Amirch80** [reply]
In Livewire 2, I could postpone the update by sending the value true in the last parameter, but this possibility is not available in Livewire 3.

What is the alternative for @this.set('prop', value, true) in livewire 3 ?

**Reply by jj15asmr** [reply]
You should be able to achieve this by setting the last parameter to ``false`` now, which tells Livewire that the update to the property shouldn't be done "live" but instead deferred.

As per [the v3 docs](https://livewire.laravel.com/docs/properties#manipulating-properties):

> In order to update the property without sending a network request to the server, you can pass a third bool parameter. This will defer the network request and on a subsequent request, the state will be synchronized on the server-side:
